### ISSUE TITLE
mcwimpy #4

### ISSUE THREAD

**Issue by mcwimpy** [Tobi823/ffupdater#439]
Stacktrace:
```
de.marmaro.krt.ffupdater.background.BackgroundException: The background job fails due to an unrecoverable exception.
	at de.marmaro.krt.ffupdater.BackgroundJob.handleDoWorkException(Unknown Source:52)
	at de.marmaro.krt.ffupdater.BackgroundJob.access$handleDoWorkException(Unknown Source:0)
	at de.marmaro.krt.ffupdater.BackgroundJob$doWork$2.invokeSuspend(Unknown Source:54)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(Unknown Source:11)
	at kotlinx.coroutines.DispatchedTask.run(Unknown Source:88)
	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(Unknown Source:0)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(Unknown Source:14)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(Unknown Source:28)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(Unknown Source:0)
Caused by: kotlinx.coroutines.JobCancellationException: Job was cancelled; job=JobImpl{Cancelling}@f099f86
```
Device information:
| Key | Value |
| --- | --- |
| FFUpdater version | 78.0.1 (143) release |
| Device | FP2 (FP2, FP2, FP2) |
| Manufacturer | Fairphone (Fairphone) | 
| Supported ABIs | armeabi-v7a, armeabi |
| Android version | 11 (SDK: 30) |
| OS | ae4c9429aa83, root, release-keys, 1687240654000 |

**Comment by Tobi823** [comment]
Is fixed in version 78.0.2 (but F-Droid is a big slow)

**Comment by mcwimpy** [comment]
Thank you
